Library/Homebrew/cask/audit.rb

```diff
@@ -597,31 +597,29 @@ def livecheck_min_os
       return unless cask.livecheckable?
       return if cask.livecheck.strategy != :sparkle
 
-      out, _, status = curl_output("--fail", "--silent", "--location", cask.livecheck.url)
-      return unless status.success?
+      # `Sparkle` strategy blocks that use the `items` argument (instead of
+      # `item`) contain arbitrary logic that ignores/overrides the strategy's
+      # sorting, so we can't identify which item would be first/newest here.
+      return if cask.livecheck.strategy_block.present? &&
+                cask.livecheck.strategy_block.parameters[0] == [:opt, :items]
```

Comment:
This seems to know way too much about the internals of this strategy block and would be better placed with something like a `cannot_do_this?` method.

Reply:
I'm not sure that I follow. Could you explain a bit more?

For context, `Sparkle` `strategy` blocks that use the `items` argument will categorically do something instead of or in addition to the default sorting. The usual argument is `item` (i.e., the first item after sorting/filtering) and we only use `items` when there's some issue where the first-sorted item isn't the newest version (e.g., an older version may have a more recent `pubDate`).